### PR TITLE
New question for FAQ for PoS voting

### LINKED ISSUE #822: Instructions what to do if VSP goes down
Inspired by [this chat](https://matrix.to/#/!MgQoetFiyjrHAywokv:decred.org/$154870767019904ThRXq:decred.org), it may be useful to have a page or section that gives VSP user a plan what to do if the VSP goes down.

### PR DESCRIPTION
**Pull request by s-ben**
Adds a question to the PoS Voting General FAQ ( Closes #822 ).

#### 14. What happens if my Voting Service Provider (VSP) goes down?¶
Voting rights are based on 1-of-2 multisig transactions. Therefore, in the unlikely event your Voting Service Provider (VSP) goes down permanently, you can still vote and receive your PoS reward, as well as revoke a missing or expired ticket. To do this, you will need to have a wallet with your VSPs pool script imported. See the [Solo Proof-of-Stake (PoS) Voting](../../advanced/solo-proof-of-stake-voting.md) page for details on setting up a wallet for solo voting.

### CODE REVIEW

**Comment by s-ben** [comment]
@jrick is this correct?

**Comment by s-ben** [comment]
Right. I was thinking most of the people reading this would not actually need to do this, but just want to know generally what would happen in this situation. I suppose eventually someone will be in this situation though and perhaps want more guidance. Perhaps just direct people to the #support channel on Slack? That really is the best move, as our devs are really good at handling complex issues there.

**Comment by jrick** [comment]
> Voting rights are based on 1-of-2 multisig transactions

I would say: "Voting rights are assigned to a 1-of-2 multisig address, where one of the required keys belongs to your wallet."

> as well as revoke a missing

missed

> To do this, you will need to have a wallet with your VSPs pool script imported.

Lots of users are using decrediton and they will only be able to revoke the ticket, not vote.  For all functionality, this needs to make it clear that they must set up a dcrwallet voting server.

**Comment by jholdstock** [comment]
Could we possibly prefix with this answer with a reassurance along the lines of "Don't worry, your funds are not lost forever."

If somebody is in a position where they really have lost their VSP, they could be legit concerned about losing their Decred permanently. I think it would be good to reassure them in laymans terms that they are safe before talking about multisig addresses and importing scripts. These technical details may not provide any comfort to a less technical user.

**Comment by s-ben** [comment]
So, users can revoke their ticket with just Decredition? They don't have to do any additional configuration? Updated answer (below) assuming this is true, and using language that doesn't freak the user out. Also, realized that this makes more sense in the VSP FAQ subpage, so have moved it there. 

#### 8. What happens if my Voting Service Provider (VSP) goes down?

If your VSP goes down, you can still vote your tickets, as well as revoke a missed or expired ticket. In this highly unlikely scenario, you can revoke your tickets in Decrediton, releasing the DCR used to purchase tickets back to your wallet. You can also vote and recieve the PoS reward if you have your VSP's pool script (available on the VSP's website). Voting rights are assigned to a 1-of-2 multisig address, where one of the required keys belongs to your wallet. Therefore, you can configure your wallet with your VSP's pool script imported, set it up for solo voting, and essentially become your own VSP. See the [Solo Proof-of-Stake (PoS) Voting](../../advanced/solo-proof-of-stake-voting.md) page for details on setting up a wallet for solo voting.

**Comment by jrick** [comment]
> "Don't worry, your funds are not lost forever."

Unfortunately, we currently can't always promise this.  If the voting script is unknown, the funds will be stuck in the vote and can not be revoked.  Users should be encouraged to backup their voting scripts, until we fix this issue by deriving both voting keys from a user's wallet seed (decred/dcrwallet#1217).

**Comment by s-ben** [comment]
How about this wording?

#### 8. What happens if my Voting Service Provider (VSP) goes down?

In the unlikely scenario your VSP goes down permanently, you can still vote your tickets, as well as revoke a missed or expired ticket. Voting rights are assigned to a 1-of-2 multisig address, where one of the required keys belongs to your wallet. This allows you to configure a wallet with your VSP's pool script imported and set it up for solo voting, essentially becoming your own VSP. This will allow you to vote or revoke tickets. Note that the VSP's pool script is necessary to perform these actions. For this reason, it is important to always keep a backup of your VSP's pool script, which can be found on your VSPs webpage. For details on setting up a wallet for solo voting, see the [Solo Proof-of-Stake (PoS) Voting](../../advanced/solo-proof-of-stake-voting.md) page.

**Comment by xaur** [comment]
> Perhaps just direct people to the #support channel on Slack?

Let's not promote Slack as a standard policy. Either Matrix or decred.org/community.

**Comment by s-ben** [comment]
@xaur , the latest doesn't mention Slack, but yeah, agree on promoting Matrix, decred.org/community generally. Viva la revolution!
